peps/pep-0766.rst

```diff
@@ -0,0 +1,444 @@
+PEP: 766
+Title: Explicit Priority Choices Among Multiple Indexes
+Author: Michael Sarahan <msarahan@gmail.com>
+Sponsor: Barry Warsaw <barry@python.org>
+PEP-Delegate: Paul Moore <p.f.moore@gmail.com>
+Discussions-To: https://discuss.python.org/t/pep-for-handling-multiple-indexes-index-priority/71589
+Status: Draft
+Type: Informational
+Topic: Packaging
+Created: 18-Nov-2024
+Post-History: `18-Nov-2024 <https://discuss.python.org/t/pep-for-handling-multiple-indexes-index-priority/71589>`__,
+
+Abstract
+========
+
+Package resolution is a key part of the Python user experience as the means of
+extending Python's core functionality. The experience of package resolution is
+mostly taken for granted until someone encounters a situation where the package
+installer does something they don't expect.  The installer behavior with
+multiple indexes has been `a common source of unexpected behavior
+<https://github.com/pypa/pip/issues/8606>`__.  Through its ubiquity, pip has
+long defined the standard expected behavior across other tools in the ecosystem,
+but Python installers are diverging with respect to how they handle multiple
+indexes. At the core of this divergence is whether index contents are combined
+before resolving distributions, or each index is handled individually in order.
+Pip merges all indexes before matching distributions, while uv matches
```

Comment:
pip is usually always lowercase, even at the start of a sentence: https://pip.pypa.io
```suggestion
pip merges all indexes before matching distributions, while uv matches
```